### PR TITLE
additional solo event and datetime field

### PR DESCRIPTION
**Pull request by tugrul**
I added a trigger for solo event and additional fields datetime and dvr_name

### CODE REVIEW

**Comment by antonsviridenko** [comment]
What does "solo" mean in this context, Bluecherry hardware capture card based on solo6x10 chips or something other?

It would be nice to have more human-readable strings instead of `array('motion_event', 'device_state', 'solo');` displayed to user.

And I missed that part in previous pull request, all strings displayed in web UI should be defined in https://github.com/bluecherrydvr/bluecherry-apps/blob/master/www/lib/lang.php instead of hardcoded literals in the code. We might need translations to other languages than English. 

https://github.com/bluecherrydvr/bluecherry-apps/blob/master/www/template/main_admin.php#L162

File: www/ajax/webhook.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function getData()
 
         $this->view->cameras = data::getObject('Devices');
         $this->view->webhooks = data::getObject('webhooks');
-        $this->view->events = array('motion_event', 'device_state'/*, 'solo'*/);
+        $this->view->events = array('motion_event', 'device_state', 'solo');
```
Comment (antonsviridenko) [inline www/ajax/webhook.php:18]: see previous comment about literal strings displayed to user

**Review by antonsviridenko — CHANGES_REQUESTED** [review]

**Comment by tugrul** [comment]
I don't know what is it. I just implemented event types what I see in mailer.php

**Comment by antonsviridenko** [comment]
Something like "Motion Event", "Device State", "Bluecherry SOLO card state"

**Comment by curtishall** [comment]
Labels:

- Motion Event
- Device Status
- Bluecherry analog capture card (solo)

**Review by antonsviridenko — APPROVED** [review]

**Pull request merged**